docs/ai-testbed/groq/groqview.md

```diff
@@ -0,0 +1,52 @@
+# Groqview profiler and visualizer tool. 
+
+This section covers how to remotely use the GroqView profiler and visualizer tool.
+
+## Groqview sample
+
+Groq compiles produce an accurate and detailed model of the performance of a model's execution ongroq cards. There is no need to run a model on groqcards to use groqview.<br>
+The groqview example adds the "groqview=True" parameter to the groqit call, then calls the groqview() method on the model returned by groqit.<br>
```

Comment:
why use `<br>` instead of just blank lines?

```suggestion
The GroqView example adds the "groqview=True" parameter to the `groqit` call, then calls the `groqview()` method on the model returned by `groqit`.
```